code/datums/elements/bed_tucking.dm

```diff
@@ -56,5 +58,5 @@
 /datum/element/bed_tuckable/proc/untuck(obj/item/tucked)
 	SIGNAL_HANDLER
 
-	tucked.transform = turn(tucked.transform, -rotation_degree)
+	tucked.transform = turn(matrix(), 0) // reset rotation
```

Comment:
Is `turn(matrix(), 0)` different from just `matrix()`?

And does this not also destroy scale? What happens if I have a scale transformed object that I tuck in bed?

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/70576/commits/b6632982b312d567222873f5d0d7c4035c51eade